provider/standalone/standalone_test.go

```diff
@@ -8,7 +8,52 @@ import (
 	"github.com/taskcluster/taskcluster-worker-runner/run"
 )
 
-func TestConfigureRun(t *testing.T) {
+func TestConfigureRunNoOptional(t *testing.T) {
+	runnerWorkerConfig := cfg.NewWorkerConfig()
+	runnerWorkerConfig, err := runnerWorkerConfig.Set("from-runner-cfg", true)
+	require.NoError(t, err, "setting config")
+	runnercfg := &cfg.RunnerConfig{
+		Provider: cfg.ProviderConfig{
+			ProviderType: "standalone",
+			Data: map[string]interface{}{
+				"rootURL":      "https://tc.example.com",
+				"clientID":     "testing",
+				"accessToken":  "at",
+				"workerPoolID": "w/p",
+				"workerGroup":  "wg",
+				"workerID":     "wi",
```

Comment:
would be good to add a non-string here, like `"runTasksAsCurrentUser": false,`

Reply:
Did you mean in the provisionerMetadata (which doesn't appear in this test)?  The set of items at Data is fixed and all of them are strings..